src/NodeManipulator/NullabilityColumnPropertyTypeResolver.php

```diff
@@ -31,7 +31,7 @@ public function __construct(
     ) {
     }
 
-    public function isNullable(Property $property): bool
+    public function isNullable(Property $property, bool $defaultNullableColumn): bool
     {
         $nullableExpr = $this->attributeFinder->findAttributeByClassArgByName(
```

Comment:
I myself are not sure about this (existing) code. This as it threads `#[Column(type: ...)]`, without `nullable`, as if the attribute isn't set at all. In which case it would test for the `@Column` annotation. While the existence of the attribute IMO needs to use either the configured `nullable` or, when absent, the default/fallback, and not looking for the `@Column` annotation.